tools/lsp/properties.rs

```diff
@@ -118,14 +117,14 @@ fn find_expression_range(
                     .first_child()
                     .expect("A BindingExpression needs to have a child!")
                     .text_range();
-                result.expression_start = expr_range.start().into();
-                result.expression_end = expr_range.end().into();
+                expression_start = expr_range.start().into();
```

Comment:
idea: instead of serializing this to u32 here, keep it as a range, and add a helper function that converts from that range to `lsp_types::Range`

there is already such a helper function `util::to_range` in util.rs,  but that's not quite the same

Reply:
Excellent idea, I'll do that!